g11n-ws/vip-common/src/main/java/com/vmware/vip/common/utils/LocaleUtils.java

```diff
@@ -27,12 +25,12 @@ public class LocaleUtils {
 	 * @return true if the format is fine
 	 */
 	public static boolean isLanguageTag(String languageTag) {
-		ParseStatus sts = new ParseStatus();
-		LanguageTag.parse(languageTag, sts);
-		if (sts.isError()) {
+		try {
```

Comment:
Why is this changed needed?

Reply:
it use the other module of jdk11, when start  JAVA VM